app/javascript/components/community-solutions/CommentsList.tsx

```diff
@@ -41,11 +42,12 @@ export const CommentsList = ({
       <Header
```

Comment:
I think add the guard for `isGuest` here, rather than in the Deader, then we don't have to pass it into Header?